src/rmDefinitionAggregation.js

```diff
@@ -5,14 +5,16 @@ const QueryURLTemplate =  "/recordm/recordm/definitions/search?"
 const ResultsURLTemplate = "#/definitions/__DEF_ID__/q=__QUERY__"
 
 const rmDefinitionAggregation = function (def, aggregation, query="*", from=0, size=10, sort="", ascending="asc", timezone) {
-  
+ 
+  let tz = timezone === null || timezone === undefined ? Intl.DateTimeFormat().resolvedOptions().timeZone : timezone
```

Comment:
`let tz = timezone ?? Intl.DateTimeFormat().resolvedOptions().timeZone` ?

Reply:
I'd like that, but the node version we use does not yet have that symbol 🙃

Reply:
Não podias simplificar isto como `!timezone ? Intl.DateTimeFormat().resolvedOptions().timeZone : timezone`